pkg/controller/controller.go

```diff
@@ -21,6 +21,8 @@ import (
 	"fmt"
 	"time"
 
+	features "sigs.k8s.io/external-resizer/pkg/features" /* TO BE REPLACED WITH features "github.com/external-resizer/pkg/features once PR merged*/
```

Comment:
Why are we importing via `sigs.k8s.io`? I do not understand reason for doing this. you should be able to import it normally via `github.com/kubernetes-csi/external-resizer`.

Reply:
Done 👍